src/components/AppRouter.js

```diff
@@ -18,7 +18,7 @@ function AppRouter() {
           <Route path="/login" component={LogIn} />
           <Route path="/logout" component={LogOut} />
           <Route path="/code-of-conduct" component={CodeOfConduct} />
-          <Route path="/me" component={Profile} />
+          <Route path="/me" render={() => (<Redirect to='/login' replace />)} />
```

Comment:
`<Redirect from="/me" to="/login" replace />` ?

Reply:
This replaces `/me` in the browser history with `/login` so that going back doesn't hit `/me` again and do that annoying thing where you keep getting redirected back to your current page.